locations/spiders/sdek.py

```diff
@@ -0,0 +1,126 @@
+from urllib.parse import urljoin
+import scrapy
+from scrapy.http import JsonRequest
+from locations.categories import Categories, PaymentMethods, apply_category, apply_yes_no
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+PAYMENT_MAPPING = {
+    "CASH": PaymentMethods.CASH,
+    "HAVE_TERMINAL": PaymentMethods.CARDS,
+    "HAVE_CASHLESS": None,
+}
+
+
+class SdekSpider(scrapy.Spider):
+    name = "sdek"
```

Comment:
Rename spider to `SDEKRUSpider` and change the name to `sdek_ru`?

Reply:
Or are there locations outside Russia?

Reply:
There are locations outside RU as well.